packages/next/build/webpack/loaders/next-middleware-ssr-loader/render.ts

```diff
@@ -1,4 +1,13 @@
+import escapeRegexp from 'next/dist/compiled/escape-string-regexp'
+
```

Comment:
```suggestion
```

Reply:
This one is intended, to separate out different types of import.